src/modules/ethereum/transactions/gasPrices/EIP1559.ts

```diff
@@ -1,8 +1,4 @@
-export interface EIP1559 {
-  estimation_seconds?: number;
-  max_fee: number;
-  priority_fee: number;
-}
+import type { EIP1559 } from '@zeriontech/transactions';
```

Comment:
Should `@zeriontech/transactions` be updated?

Reply:
It should. I'm waiting for the release :)